hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaAction.java

```diff
@@ -126,6 +126,9 @@
 
     @Column(name = "maintenance_time_zone", updatable = false, length = Action.MAINTENANCE_WINDOW_TIMEZONE_LENGTH)
     private String maintenanceWindowTimeZone;
+    
+    @Column(name = "external_ref", updatable = false, length = Action.EXTERNAL_REF_LENGTH)
```

Comment:
why `updatable = false`? don't we want to make it updatable?

Reply:
I was under the impression that updatable=false does allow us to update the value once already set. I was looking for an option where externalRef cannot be over written.